tasks/transferErc20.ts

```diff
@@ -0,0 +1,66 @@
+import { task } from 'hardhat/config'
+import erc20abi from '../abi/IERC20.json'
+
+task('transferErc20', 'Transfers erc20 funds between two addresses.  Mainnet not supported :(')
+  .addParam('from', 'Address of the wallet that we are transferring tokens from (robbing the rich)')
+  .addParam('to', 'Address of the wallet that we are transferring tokens to (to feed the poor)')
+  .addParam(
+    'token',
+    'The address of the token that we want to transfer.  The entire balance at theRich address will be transferred to thePoor.',
```

Comment:
you have left `thePoor` and `theRich` in param description.

Reply:
@paszkowskiDamian addressed here: https://github.com/OasisDEX/multiply-proxy-actions/pull/116